api/main.go

```diff
@@ -105,6 +104,15 @@ func main() {
 		routes = append(routes, controllers.NewTagController(tagStore).Routes()...)
 		routes = append(routes, controllers.NewTaskController(taskProvider, jobStore, tagStore).Routes()...)
 
+		adminRoutes := controllers.NewAdminController(cfg, Version).Routes()
```

Comment:
This can be simplified:  just make a new endpoint, `/health` with a different controller; there is no need for the health check to be tied to the `admin` controller. Then, `append` the health route to `routes` after the routes have been decorated.

Reply:
Ah! Sure thing. That should be straightforward enough.